api/src/main/java/org/bf2/operator/resources/v1alpha1/Versions.java

```diff
@@ -29,35 +31,14 @@ public class Versions {
     public static final String STRIMZI_CLUSTER_OPERATOR_V0_23_0_4 = "strimzi-cluster-operator.v0.23.0-4";
 
     @NotNull
+    @Getter @Setter
```

Comment:
What happens with these on the class instead of being on the fields?

Reply:
It generates for only those fields. But I think I can move it to class level here as well since the other variables are static and final.